controllers/grid/submissions/SubmissionsListGridCellProvider.inc.php

```diff
@@ -64,20 +64,52 @@ function getCellState($row, $column) {
 	 * @return array an array of LinkAction instances
 	 */
 	function getCellActions($request, $row, $column, $position = GRID_ACTION_POSITION_DEFAULT) {
-		if ( $column->getId() == 'title' ) {
-			$submission = $row->getData();
+		$submission = $row->getData();
+		if ($column->getId() == 'editor') {
+			$stageAssignmentDao = DAORegistry::getDAO('StageAssignmentDAO'); /* @var $stageAssignmentDao StageAssignmentDAO */
+			$editorAssignments = $stageAssignmentDao->getEditorsAssignedToStage($submission->getId(), $submission->getStageId());
+			$assignment = current($editorAssignments);
+			if (!$assignment) return array();
+			$user = $request->getUser();
+			$userDao = DAORegistry::getDAO('UserDAO'); /* @var $userDao UserDAO */
+			$editor = $userDao->getById($assignment->getUserId());
```

Comment:
OTOH, I think the EditAssignment object contains enough identifying information that the User object fetch might be unnecessary.